docs/lang/articles/reference.md

```diff
@@ -198,6 +198,24 @@ compile time.
 
 ### Conditional expressions
 
+### Static expressions
+
+```
+static_expression ::= "ti.static(" positional_arguments ")"
+```
+
+Static expressions are expressions that are wrapped by a call to `ti.static()`.
+The `positional_arguments` is evaluated at compile time, and the variables inside it must be compile-time constants.
```

Comment:
> and the variables inside it must be compile-time constants

I think it's more like these variables are Python objects?

Reply:
Yes. Let's unify the terms and say "and the items inside must be evaluated to Python values".